src/components/map/mapactions/custom-map.js

```diff
@@ -115,6 +115,10 @@ export class VlCustomMap extends VlMapWithActions {
 
   addBaseLayerAndOverlayMapLayer(baseLayer, overlayMapLayer) {
     baseLayer.setVisible(this.baseLayers.length === 0);
+    overlayMapLayer.setVisible(
+        !this.overviewMapControl ||
+        this.overviewMapControl.getOverviewMap().getLayers().getArray().length === 0
+    );
```

Comment:
Deze change is gebeurt zodat de overlaymap direct rendert indien het de eerste is.